recipes-kernel/linux/linux-mainline_%.bbappend

```diff
@@ -17,6 +17,7 @@ SRC_URI_append = " \
         file://usbnet.cfg \
         file://usbserial.cfg \
         file://filesystems.cfg \
+	file://lima.cfg \
```

Comment:
I'd call it `graphics.cfg` - most of the options doesn't seem to be directly related to lima driver.

Reply:
Renamed.